.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: CI
+on:
+  pull_request:
+
+jobs:
+  airgapped_build:
+    name: Airgapped build
+    runs-on: ubuntu-20.04
```

Comment:
Can you add a if condition for PR only just in case that we expand the workflow to cover push?

Reply:
Yes, but now I think about it does this need to be PR-only?

Reply:
Yeah, probably not.